src/lib.rs

```diff
@@ -505,6 +622,7 @@ where
         CompactFormatBuilder {
             fn_timestamp: Box::new(timestamp_local),
             decorator: d,
+            header_printer: None,
```

Comment:
Did you miss this one, or am I missing something?

Reply:
Sorry my bad, totally forgot the 2nd one ^^'